core/storage/skill/gae_models.py

```diff
@@ -43,7 +43,7 @@ class SkillModel(base_models.VersionedModel):
 
     # The description of the skill.
     description = ndb.StringProperty(required=True, indexed=True)
-    # The schema version for each of the misconceptions dict.
+    # The schema version for each of the misconceptions dicts.
```

Comment:
sorry, should be misconception dicts (this is a list of dicts each representing a single misconception, right?)

Reply:
Done.